pages/common/kosmorro.md

```diff
@@ -0,0 +1,20 @@
+# kosmorro
+
+> Compute the ephemerides and the events for a given date, at a given position on Earth.
+> More information: <http://kosmorro.space>.
+
+- Get ephemerides for Paris, France:
+
+`kosmorro --latitude={{48.7996}} --longitude={{2.3511}}`
+
+- Get ephemerides for Paris, France, in the UTC+2 timezone:
+
+`kosmorro --latitude={{48.7996}} --longitude={{2.3511}} --timezone={{2}}`
+
+- Get ephemerides for Paris, France, on June 9th, 2020:
+
+`kosmorro --latitude={{48.7996}} --longitude={{2.3511}} --date={{2020-06-09}}`
+
+- Generate a PDF (note: you need to install TeXLive):
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a PDF (note: TeXLive must be installed):
```

Generally we try to avoid the 2nd person (e..g `you`, `your`, etc) in tldr pages.